pkg/operators/v1/olmconfig_types.go

```diff
@@ -23,6 +25,16 @@ type Features struct {
 	// When reenabled, OLM will recreate the "Copied CSVs" for each
 	// cluster scoped operator.
 	DisableCopiedCSVs *bool `json:"disableCopiedCSVs,omitempty"`
+	// PackageServerSyncInterval is used to define the sync interval for
+	// packagerserver pods. Packageserver pods periodically check the
+	// status of CatalogSources; this specifies the period using duration
+	// format (e.g. "60m"). For this parameter, only hours ("h"), minutes
+	// ("m"), and seconds ("s") may be specified. When not specified, the
+	// period defaults to the value specified within the packageserver.
+	// +optional
+	// +kubebuilder:validation:Type=string
+	// +kubebuilder:validation:Pattern="^([0-9]+(\\.[0-9]+)?(s|m|h))+$"
```

Comment:
For this part of the regex, `(\\.[0-9]+)?`, it matches a literal `\`, then any character, then a digit; 0 or 1 time. This means that for the regex as a whole, `1\a3m` matches. I'm guessing this isn't intended?

Reply:
It doesn't match the literal `\`... the double slash is in there because it's a comment, and thus the escape for the `.` needs to be escaped itself.
I pulled it from https://github.com/openshift/hive/pull/1684 
see: https://github.com/openshift/hive/blob/227c25916da4c0b1bb3255b7c030d2ccf36384a0/apis/hive/v1/clusterdeployment_types.go#L187

Reply:
Don't ask how I ended up here ... but FYI @tmshort if you use back-tick quotes you don't need to do the escaping:

```
// +kubebuilder:validation:Pattern=`^([0-9]+(\.[0-9]+)?(s|m|h))+$`
```